koku/masu/api/urls.py

```diff
@@ -49,6 +50,7 @@
     path("report_data/", report_data, name="report_data"),
     path("source_cleanup/", cleanup, name="cleanup"),
     path("trino/query/", trino_query, name="trino_query"),
+    path("trino_ui/", trino_ui, name="trino_ui"),
```

Comment:
```suggestion
    path("trino/api/", trino_ui, name="trino_ui"),
```

Let's use a `/` instead of `_` for this

Reply:
is this  `/` vs `_`  for consistency with internal trino endpoints?